CHANGELOG.md

```diff
@@ -1,17 +1,25 @@
 # nf-core/tools: Changelog
 
-## v2.4dev
+## [v2.3.1 - Mercury Vulture Formatting](https://github.com/nf-core/tools/releases/tag/2.3.1) - [2022-03-23]
 
 ### Template
 
-- Replace `markdownlint` and `yamllint` with [_Prettier_](https://prettier.io) for linting formatting / whitespace.
+- Replace `markdownlint` and `yamllint` with [_Prettier_](https://prettier.io) for linting formatting / whitespace ([#1470](https://github.com/nf-core/tools/pull/1470))
```

Comment:
should we add a bit more info on how to use it?

Reply:
Yeah I was going to write some more in the release changelog, but I guess I could write it here as well...